name.abuchen.portfolio.bootstrap/Application.e4xmi

```diff
@@ -134,8 +134,11 @@
         <children xsi:type="menu:HandledMenuItem" xmi:id="_mRCKILOiEeWe1a5u3oOf1Q" elementId="name.abuchen.portfolio.ui.menu.help.wiki" label="Forum" tooltip="%command.openbrowser.forum.tooltip" command="_dYDXQLOiEeWe1a5u3oOf1Q">
           <parameters xmi:id="_pMDJwLOiEeWe1a5u3oOf1Q" elementId="" name="name.abuchen.portfolio.ui.param.url" value="https://forum.portfolio-performance.info"/>
         </children>
-        <children xsi:type="menu:HandledMenuItem" xmi:id="_YUNIsJPVEe2-04kloZOp3A" elementId="name.abuchen.portfolio.ui.menu.help.wiki" label="%command.openbrowser.translationTeams.name" iconURI="platform:/plugin/name.abuchen.portfolio.ui/icons/poeditor-logo.png" tooltip="%command.openbrowser.translationTeams.tooltip" command="_dYDXQLOiEeWe1a5u3oOf1Q">
```

Comment:
The elementId should be unique to my understanding.

And: the diff would be easier to read if the old entry kept the ids (instead of giving the old entry new ids and using the old ids for the new entry).

I will update the git commit and merge. Thanks